### ISSUE TITLE
Preserve child organizations when importing?

### ISSUE THREAD

**Issue by deniszgonjanin** [mheadd/communitydata.io#18]
Right now the harvester puts datasets in organizations based on where the dataset was imported. 

Eg. http://communitydata.io/organization/open-colorado has all datasets from http://data.opencolorado.org/. But that site has it's own organizations structure, which is then lost.

If preserving organizations is desirable, it could possibly be done using https://github.com/datagovuk/ckanext-hierarchy. 

This way all organizations from imported catalogs could be placed under the parent organization, preserving each catalogs structure.

**Comment by mheadd** [comment]
Yeah, I was thinking much the same thing when looking at the [Open Data Philly Site](https://www.opendataphilly.org/organization) - it uses the same approach as Open Colorado.
